imports/ui/components/news/news_list.js

```diff
@@ -2,33 +2,33 @@ import './news_list.html';
 import { Meteor } from 'meteor/meteor';
 import { Template } from 'meteor/templating';
 import { Posts } from '../../../api/posts/posts_collection.js';
-import { FlowRouter } from 'meteor/kadira:flow-router';
+import { FlowRouter } from 'meteor/ostrio:flow-router-extra';
 
 Template.newsList.onCreated(function () {
   var self = this;
-  self.pagination = new Meteor.Pagination(Posts, { sort: { showDate: -1 }, perPage: 8, filters: {approved: true}});
+  self.pagination = new Meteor.Pagination(Posts, { sort: { showDate: -1 }, perPage: 4, filters: {approved: true}});
 });
 
 Template.newsList.onRendered(function () {
   Session.set('documentTitle', 'KTUH Honolulu | Radio For The People');
-    window.setTimeout(function() {
+    /* window.setTimeout(function() {
     $('.news-list').imagesLoaded(function() {
       $('.news-list').masonry({
         itemSelector: '.news-list__post',
         transitionDuration: 0,
         isResizeBound: true
       });
     });
-  }, 500);
+  }, 500); */
```

Comment:
IIRC Nicki's design doesn't use Masonry on this page, so we should delete this altogether.